### DISCUSSION TITLE
Does wallet-core support signing transaction token on BEP20 or ERC20?

### DISCUSSION THREAD

**Discussion by Raden-Hor** [trustwallet/wallet-core#2005] [Q&A] [answered]
Does wallet-core support signing transaction token on BEP20 or ERC20?

**Comment by optout21** [comment, marked as answer]
Yes, both.
Some references:
https://github.com/trustwallet/wallet-core/blob/master/tests/BinanceSmartChain/SignerTests.cpp#L48
https://github.com/trustwallet/wallet-core/blob/master/tests/Ethereum/SignerTests.cpp#L71
https://github.com/trustwallet/wallet-core/blob/master/tests/Ethereum/TWAnySignerTests.cpp#L79

**Reply by Raden-Hor** [reply]
Thank @catenocrypt one more question how trust wallet estimate fee or gasfee on every network that wallet-core support?

**Reply by hewigovens** [reply]
You need to do your own research for those basic questions

**Reply by Raden-Hor** [reply]
okay
